cid/builtin/core/data/resources.yaml

```diff
@@ -2,7 +2,9 @@
 # QuickSight Dashboards definitions
 dashboards:
   CUDOS:
+    category: 'Foundational'
     name: CUDOS Dashboard
+    deprecationNotice: ' DEPRECATED. Replaced with CUDOS Dashboard v5.'
```

Comment:
Maybe let's give this param more general purpose name like description?

Reply:
Description can be a separate feature. deprecationNotice is used to detect if it is deprecated or not